vms/platformvm/txs/builder/builder.go

```diff
@@ -480,6 +521,70 @@ func (b *builder) NewCreateSubnetTx(
 	return tx, tx.SyntacticVerify(b.ctx)
 }
 
+func (b *builder) NewTransformSubnetTx(
+	subnetID ids.ID,
+	assetID ids.ID,
+	initialSupply uint64,
+	maxSupply uint64,
+	minConsumptionRate uint64,
+	maxConsumptionRate uint64,
+	minValidatorStake uint64,
+	maxValidatorStake uint64,
+	minStakeDuration time.Duration,
+	maxStakeDuration time.Duration,
+	minDelegationFee uint32,
+	minDelegatorStake uint64,
+	maxValidatorWeightFactor byte,
+	uptimeRequirement uint32,
+	keys []*secp256k1.PrivateKey,
+	changeAddr ids.ShortID,
+	memo []byte,
+) (*txs.Tx, error) {
+	ins, unstakedOuts, _, signers, err := b.Spend(b.state, keys, 0, b.cfg.TransformSubnetTxFee, changeAddr)
```

Comment:
I think we typically just call these `outs` unless we are distinguishing between staked and unstaked outputs.
```suggestion
	ins, outs, _, signers, err := b.Spend(b.state, keys, 0, b.cfg.TransformSubnetTxFee, changeAddr)
```

Reply:
Done